test/postgresql-contrib.bats

```diff
@@ -10,7 +10,7 @@ contrib-only() {
 
 @test "It should support PLV8 1.4.4" {
   contrib-only
-  versions-only le 9.4
+  versions-only le 9.5
```

Comment:
I believe since this is `le` you'll want to scrap the test entirely?  Rather than `ge` which changes to 9.5.